src/Tribe/Adjacent_Events.php

```diff
@@ -0,0 +1,329 @@
+<?php
+/**
+ * Controls getting a previous or next event from the context of a single event being viewed.
+ */
+
+// Don't load directly
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+if ( ! class_exists( 'Tribe__Events__Adjacent_Events' ) ) {
```

Comment:
No need to include this class check.

Reply:
many of our files have this, so was just copying their approach — any reason for why some have it and some don't?

Reply:
It's a bad behavior that we had in the past and we want to avoid doing it going foward